### PR TITLE
Fix a bunch of errors in autocapitalize

### PR DESCRIPTION
**Pull request by dandv**

### CODE REVIEW

File: src/content/en/updates/posts/2015/04/autocapitalize.markdown [hunk 1]
```diff
@@ -20,26 +20,26 @@ Notice everything is in lowercase apart from some values that Android recognised
...
 
-A simple example letting you autocapitalize sentences in a text area.
+Here's a simple example letting you autocapitalize sentences in a text area:
 
 `<textarea autocapitalize="sentences">`
 
 [[Demo]]
 
-## What values can autocapitalize take?
+## What values can _autocapitalize_ take?
```
Comment (jpmedley) [inline src/content/en/updates/posts/2015/04/autocapitalize.markdown]: Don't use character styles in headings. Please remove.

File: src/content/en/updates/posts/2015/04/autocapitalize.markdown [hunk 2]
```diff
@@ -100,39 +95,43 @@ Capitalization_. This is a change from the default behaviour.
...
-`autocapitalize` is used across Millions of pages on hundred of thousands of
+`autocapitalize` is used across millions of pages on hundred of thousands of
 websites already.
 
 ## When should I use this?
 
-This isn't an exhaustive list of when you should use \`autocapitalize\`, however
-there a number of places where helping the user enter text helps a great deal.
+This isn't an exhaustive list of when you should use `autocapitalize`; however
+there a number of places where aiding the user enter text helps a great deal:
```
Comment (jpmedley) [inline src/content/en/updates/posts/2015/04/autocapitalize.markdown]: should be "aiding the user who is entering text" 

I'm pointing out that there's a faulty verb after your change.

**Comment by jpmedley** [comment]
@dandv,

I found a couple of things that need to be fixed before we merge this. Can you look at them real quick.

Joe

**Comment by dandv** [comment]
Hey @jpmedley, sorry it took so long, I was traveling to the NYC IWD PWA workshop. All done now.

**Pull request merged**